andela/AndelaEatsUI
# #160553937 Admin view meal items (Functionality)

#### What does this PR do?
- Adds functionality for viewing meal items on the meal page of the admin dashboard
#### Description of Task to be completed?
- Fetch and replace static contents with dynamic contents
#### How should this be manually tested?
- Log in to the app
- Switch to the admin dashboard from the user profile drop-down menu or [here](http://andelaeats-dev.andela.com:3000/admin/meals)
- You should see orders displayed
#### What are the relevant pivotal tracker stories?
- [ #160553937](https://www.pivotaltracker.com/story/show/160553937)
#### Background Context
The meal items currently being fetched are resourced using the apiary mock server for now until the backend functionality for the orders endpoint is made available
#### Screenshots
![image](https://user-images.githubusercontent.com/26303683/45837109-71b5e680-bd06-11e8-8933-ed448546913d.png)
![image](https://user-images.githubusercontent.com/26303683/45837174-9dd16780-bd06-11e8-8c87-634c69deec67.png)
